src/lib/dataDog.test.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  accountListIdsStorageKey,
+  clearDataDogUser,
+  isDataDogConfigured,
+  setDataDogUser,
+} from './dataDog';
+
+const setDataDogUserMock = {
+  userId: '123456',
+  accountListId: '1234-4567-8910-1112-1314',
+  name: 'Roger',
+  email: 'roger@cru.org',
+};
+
+describe('dataDog', () => {
+  beforeEach(() => {
+    window.DD_RUM = {
+      setUser: jest.fn(),
+      clearUser: jest.fn(),
+    };
+  });
+
+  describe('when DataDog is not configured', () => {
+    it('isDataDogConfigured should return false', () => {
+      expect(isDataDogConfigured()).toEqual(false);
+    });
+
+    it('setDataDogUser should not call DD_RUM methods', () => {
+      setDataDogUser(setDataDogUserMock);
+      expect(window.DD_RUM.clearUser).not.toHaveBeenCalled();
+      expect(window.DD_RUM.setUser).not.toHaveBeenCalled();
+    });
+  });
+
+  describe('when DataDog is configured', () => {
+    beforeEach(() => {
+      process.env.DATADOG_CONFIGURED = 'true';
+    });
+
+    //#region Default Tests
+    it('isDataDogConfigured should return false', () => {
```

Comment:
```suggestion
    it('isDataDogConfigured should return true', () => {
```

Reply:
Good catch, thanks! I must have accidentally changed that.